deeplabcut/pose_estimation_tensorflow/nnet/predict.py

```diff
@@ -71,6 +74,23 @@ def argmax_pose_predict(scmap, offmat, stride):
                                [scmap[maxloc][joint_idx]])))
     return np.array(pose)
 
+
```

Comment:
This function is defined twice --> See two methods below

Reply:
The is intended as I suggested before I still don't know how this feature will be integrated into the DLC. For your testing purposes use https://github.com/ehsainit/DeepLabCut/blob/c34b849999a497f81675f9c27ef4d212e71c7183/deeplabcut/pose_estimation_tensorflow/nnet/predict.py#L78